drivers/sensor/si1153/Kconfig

```diff
@@ -0,0 +1,77 @@
+# Kconfig.si1153 - SI1153 Proximity/Ambient Light Sensor IC with I2C Interface
+
+#
+# Copyright (c) 2017 IpTronix S.r.l.
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+
+menuconfig SI1153
+	bool "SI1153 sensor"
+	depends on SENSOR && I2C
+	default n
+	help
+	 Enable driver for SI1153 Proximity/Ambient Light Sensor IC with I2C Interface.
```

Comment:
Alignment should be:
\<tab>\<space>\<space>help text

This applies to the rest of the configs as well.